app/lib/features/comments/widgets/create_comment.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:acter/common/widgets/html_editor.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_easyloading/flutter_easyloading.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class CreateCommentWidget extends ConsumerStatefulWidget {
```

Comment:
We can convert this to simple ```Stateful```, don't see reason otherwise ;)

Reply:
editorState and whether the editor is shown (`opened`) are stateful. This is a stateful widget.